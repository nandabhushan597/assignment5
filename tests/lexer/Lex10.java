int for all
    /* this is a break comment */
  while do HI THERE // something else
	     /* blah */ blah
