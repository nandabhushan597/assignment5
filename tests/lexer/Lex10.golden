Right [KeywordT IntK
KeywordT ForK
IdentifierT "all"
KeywordT WhileK
KeywordT DoK
IdentifierT "HI"
IdentifierT "THERE"
IdentifierT "blah"]